client/src/features/location-rating/location-rating.actions.js

```diff
@@ -9,3 +10,39 @@ export function updateLocationRatingAction(location, feature) {
     }
   };
 }
+
+export function updateSuccess(bool) {
+  return {
+    type: LOCATION_RATING_TYPES.saveDbSuccess,
+    payload: {
+      bool
+    }
+  };
+}
+
+export const saveLocationToDb = (name, lat, lon, features) => {
+  return dispatch => {
+    apiService
+      .post(`/someModels/${name}`, {
+        name,
+        lat,
+        lon,
+        rating: {
+          accessible_parking: features.includes('Accessible parking'),
+          automatic_front_door: features.includes(
+            'Ramp leading to the front door'
+          ),
```

Comment:
I think these two comments - automatic_front_door and front_door_ramp needs to be swapped. 
automatic_front_door should have the message 'front door opens automatically'
and front_door_ramp 'ramp leads to the front door'